scf_literature.md

```diff
@@ -0,0 +1,177 @@
+"Starting SCF Calculations by Superposition of Atomic Densities"
+J. H. Van Lenthe, R. Zwaans, H. J. J. Van Dam, M. F. Guest, J. Comput. Chem. 2006, 27, 8, 926-932; https://doi.org/10.100
+/jcc.20393
+
```

Comment:
I hope my brief summaries weren't too reductive, though inevitably I anticipate that further details and clarifications will be requested that pertain to this document.

Reply:
These look great!! As long as we have the citations we can always consult the papers if we need more details, so I wouldn't worry about going into more detail.

Reply:
Thanks for saying so. I can look at more papers if we want/need to. What do you think?

Reply:
I'd keep digging (a great way to keep finding more papers is to consult the intro sections of the papers you've already found). SCF guesses go back to the advent of quantum chemistry so there's likely to be a lot more out there.